### ISSUE TITLE
mockResponse returning undefined

### ISSUE THREAD

**Issue by ericdfields** [jefflau/jest-fetch-mock#7]
I'm sure I'm missing something pertaining to using this lib, but fetch.mockResponse is returning 'undefined' for me. 

I've assigned `fetch` to your library in my jestsetup.js per the instructions. In my test I've just written:

```
const source = fetch.mockResponse(JSON.stringify({test: '123'}))
console.log(source)
```

`source` is undefined. What'd I miss?

Thanks

**Comment by kwelch** [comment]
There is not return value for mockResponse, this simple set the mockImplementation of the jest spy, which continues to be `fetch`.

**Comment by jefflau** [comment]
This should be fixed in 1.1
